### ISSUE TITLE
Low buffer values appear to cause the server to break

### ISSUE THREAD

**Issue by redmoogle** [lavalink-devs/Lavalink#627]
## Description
Setting `bufferDurationMs` to a low value such as `10` breaks the audio output of the client without producing any errors

This should have a note if its intentional (or maybe im dumb and I dont see it)

## Version info
Lavalink.py:master
Lavalink 3.4

Output of `java -version`:
```
openjdk 17.0.1 2021-10-19
OpenJDK Runtime Environment (build 17.0.1+12-Ubuntu-121.10)
OpenJDK 64-Bit Server VM (build 17.0.1+12-Ubuntu-121.10, mixed mode, sharing)
```

[spring.log.txt](https://github.com/freyacodes/Lavalink/files/8112926/spring.log.txt)

**Comment by freyacodes** [comment]
Not super surprising. Should probably add a lower limit

**Comment by devoxin** [comment]
Closing as this should be fixed as of `v3.5-rc2`.
